src/pages/home/ReportScreen.js

```diff
@@ -167,7 +167,10 @@ function ReportScreen({
     const reportID = getReportID(route);
 
     const isFocused = useIsFocused();
+
+    // Update the lastVisitTime when the report is focused
```

Comment:
> We need to update the `lastVisitTime` every time the report is focused because `lastReadTime` does not have up-to-date time stamp every time the user visits a report.

Lets update it to this.

Reply:
Updated

Reply:
Please help check again @getusha